dotnet/tests/Microsoft.Agents.AI.UnitTests/Data/RagProviderTests.cs

```diff
@@ -0,0 +1,503 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Agents.AI.Data;
+using Microsoft.Extensions.AI;
+using Microsoft.Extensions.Logging;
+using Moq;
+
+namespace Microsoft.Agents.AI.UnitTests.Data;
```

Comment:
Suggest renaming the folder as well, should end with `UnitTests`
```suggestion
namespace Microsoft.Agents.AI.Data.UnitTests;
```

Reply:
As decided in the scrum, this will stay in the MAAI package.